src/app/component/AddressInput/AddressInput.jsx

```diff
@@ -1,11 +1,16 @@
 import AlgoliaPlaces from "algolia-places-react";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useRef, useEffect } from "react";
 
 const { ALGOLIA_API_KEY, ALGOLIA_APP_ID } = process.env;
 
 const AddressInput = (props) => {
-  const { placeholder, setStage, stage } = props;
+  const { error, onClear, placeholder, setStage, stage, value } = props;
+  const placesRef = useRef();
```

Comment:
Adding this ref so we can set a default value in the algolia input.
Needed for when a person goes back in their checkout experience to change details